src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -697,4 +702,56 @@ public YesOrNo getRespondent2Represented() {
             .filter(Objects::nonNull)
             .findFirst().orElse(null);
     }
+
+    @JsonIgnore
+    public DefendantResponseStatus getDefendantResponseStatus() {
+        if (defenceAdmitPartPaymentTimeRouteRequired != null
```

Comment:
Would be good to move out each complex if condition here to a more descriptive method - e.g. move this to a method called isEligibleForCCJ perhaps?